src/background.ts

```diff
@@ -14,9 +14,9 @@ const players: Player[] = [
   },
   {
     controlQueries: {
-      next: ".control.control-next",
-      play: ".control.control-play",
-      prev: ".control.control-prev",
+      next: ".control.control-next, .player-controls .svg-icon-group .svg-icon-group-item:nth-of-type(5) button",
```

Comment:
I think it's probably better to remove the `.svg-icon-group`. The fewer the selector contents, the fewer the chances of having site updates breaking it.

Reply:
Same thing for the lines below